StableIPFSLoader.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="fragment" content="!">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Decentralized Prediction Markets | Augur</title>
+  <meta name="description"
+        content="Augur is a decentralized oracle and prediction market protocol built on the Ethereum blockchain. It allows you to forecast events and be rewarded for predicting them correctly.">
+  <script src="https://unpkg.com/ipfs/dist/index.min.js"></script>
+  <script>
+      (async function() {
+          const node = await window.Ipfs.create();
```

Comment:
This is fine for now but we need to see about using a single IPFS worker that all the cases in the UI communicate to (sorta like the one we have now).

It can be a separate task to combine them.